lib/rb_shift/project.rb

```diff
@@ -60,54 +60,64 @@ def routes(update = false)
     end
 
     def create_secret(name, kind, **opts)
+      log.info "Creating secret #{kind} #{name}"
       execute "create secret #{kind} #{name}", **opts
       secrets true if @_secrets
     end
 
     def add_role_to_user(role, username)
+      log.info "Adding role to user #{role} #{username}"
```

Comment:
The word order seems a bit weird.. what about "Adding role #{role} to user #{username}"